src/stories/Library/search-result-page/search-result-page.scss

```diff
@@ -11,3 +11,41 @@
     gap: 16px;
   }
 }
+
+.facet-line {
+  &__item {
+    display: inline-block;
+    @extend %mr-8;
+    @extend %mt-8;
+
+    .dropdown__select {
+      @extend %text-tags;
+      height: 100%;
+      padding: 8px 16px;
+      min-width: 200px;
+    }
+    .dropdown__arrows {
+      height: 100%;
+    }
+  }
+}
+
+.facet-line-selected {
+  @extend %mt-16;
+
+  &__item {
+    display: inline-block;
+    @extend %mr-8;
+    @extend %mt-8;
+
+    .dropdown__select {
```

Comment:
Same with these classes that don't match the component. We would like them to be modifiers to already existing dropdown__select and dropdown__arrows (if they differ from the regular ones).

Reply:
👍